mediorum/ddl/cid_lookup.sql

```diff
@@ -8,22 +8,20 @@ create table if not exists cid_lookup (
 );
 
 -- creator-node creates the Files table, but we need to create it here in case we're running via audius-compose without CNs
-CREATE TABLE IF NOT EXISTS "Files" (
-    "id" SERIAL PRIMARY KEY,
-    "ownerId" INTEGER NOT NULL,
-    "multihash" text NOT NULL,
+CREATE TABLE if not exists "Files" (
+    multihash text NOT NULL,
     "sourceFile" text,
     "storagePath" text NOT NULL,
     "createdAt" timestamp with time zone NOT NULL,
     "updatedAt" timestamp with time zone NOT NULL,
-    "fileUUID" uuid NOT NULL UNIQUE,
+    "fileUUID" uuid NOT NULL,
     "cnodeUserUUID" uuid,
-    "type" character varying(16),
+    type character varying(16),
     "fileName" text,
     "dirMultihash" text,
-    -- "trackBlockchainId" integer, -- this is part of the table, but CN migration sets it and breaks if it already exists. initdb/init.sql will create it if running without CNs
-    "clock" integer NOT NULL,
-    "skipped" boolean DEFAULT false NOT NULL
+    "trackBlockchainId" integer,
```

Comment:
flagging this for what we synced about offline. some of these issues might be alleviated by the "if not exists" changes I added to CN earlier, but still on board with your schema-as-a-first-class thing for next week